app_flutter/lib/index_page.dart

```diff
@@ -114,6 +114,17 @@ class Index extends StatelessWidget {
                 ),
               ),
               Container(height: 25),
+              SizedBox(
+                width: 300,
+                child: RaisedButton(
+                  child: const Text('Skia Perf'),
```

Comment:
Nit: "Skia perf" is a little confusing. Maybe call it "Flutter on skia perf"? Once we also imported data from cocoon benchmarks to Flutter's skia perf instance, we may also just call it "Flutter perf" and put our current "Benchmarks" in a less promising position.

Reply:
Updated to "Benchmarks on Skia Perf". Let me know when we can make it "Benchmarks" :)